src/js/askForData.js

```diff
@@ -0,0 +1,28 @@
+import getData from "./fetch.js"
+import showWheater from "./showWheater"
```

Comment:
Nie jestem pewien, czy jest sens rozdzielać każdą z tych funkcji na osobne pliki. Wszystkie te funkcje są ze sobą powiązane i mają podobną logikę i sądzę, że wystarczyłoby wszystkie razem trzymać w osobnym pliku od app.js razem.

Reply:
Hej ogólnie jedne pobierają dane aktualnej pogody, drugie dla prognozy pogody na najbliższe dni   więc są dwa rodzaje callbacków które muszą obsługiwać - poprawiłem to już

Reply:
wcześniej po prostu nie zorientowałem się że aktualna pogoda a prognoza to co innego